.travis.yml

```diff
@@ -18,38 +7,66 @@ jobs:
       osx_image: xcode11.2
       env:
         - MACOSX_DEPLOYMENT_TARGET=10.14
-        - PATH="${HOME}/Library/Python/2.7/bin:${PATH}"
         - MATRIX_EVAL=""
+      install:  # Overrides install step below
+        # pyenv doesn't exist on this CI image, so use virtualenv to set Python version
+        - pip3 install virtualenv
+        - virtualenv -p python3 ~/venv
+        - source ~/venv/bin/activate
+
     - os: linux
       dist: bionic
       compiler: gcc
       env:
-        - GCC_VER="9"
+        - GCC_VER="10"
         - MATRIX_EVAL="CC=gcc-${GCC_VER} && CXX=g++-${GCC_VER}"
 
       addons:
         apt:
           sources:
             - ubuntu-toolchain-r-test
           packages:
-            # I couldn't get ${GCC_VER} in here successfully
-            - gcc-9
-            - g++-9
+            - gcc-10
+            - g++-10
             - doxygen
-            - python3
-            - python3.8
       after_script:
         - bash <(curl -s https://codecov.io/bash) -x /usr/bin/gcov-${GCC_VER}
     - os: linux
       dist: bionic
```

Comment:
If we bump this up to 'focal' (Ubuntu 20.04), I think we get Python 3 set to the default interpreter by default.